.yamato/_projectcontext.yml

```diff
@@ -379,6 +413,19 @@ all_package_ci_project_trunk:
       -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
       -  upm-ci package izon -t
       -  upm-ci package izon -d
+all_package_ci_project_2020.2:
+    name: Pack and test all packages - 2020.2 [project context]
+    agent:
+        type: Unity::VM
+        image: package-ci/win10:stable
+        flavor: b1.large
+    dependencies:
+      -  .yamato/_projectcontext.yml#test_all_project_windows_2020.2
+      -  .yamato/_projectcontext.yml#test_all_project_macos_2020.2
+    commands:
+      -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
+      -  upm-ci package izon -t
+      -  upm-ci package izon -d
```

Comment:
i think a new job needs to be set up for publishing 2020.2 packages in project context? right now the publish job uses the trunk dependencies

Reply:
you mean to change the existing jobs' dependencies to 2020.2? or a completely new job beside the trunk dependent one?

Reply:
completely new one

Reply:
added 2020.2 dependency under the publish job, so that we test for both for now, because the publish part is the same in both cases anyways